index.html.in

```diff
@@ -74,7 +74,7 @@
 
         <div id="wrap">
           <nav class="navbar navbar-default" role="navigation">
-              <div class="container" style="position: relative">
+              <div class="container posRelative">
```

Comment:
https://getbootstrap.com/docs/4.0/utilities/position/ `position-relative` in the utilities section as discussed below.